app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb

```diff
@@ -0,0 +1,109 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">How can I fund my teacher training?</h2>
+
+    <div class="inset">
+      <p>
+        Different bursaries and scholarships are available depending on the subject you're training to teach.
+      </p>
+
+      <h3>Postgraduate bursaries and scholarships</h3>
+
+      <table class="first-column-headings right-aligned-values">
+        <caption>Postgraduate bursaries and scholarships are available for the subjects listed below.</caption>
+        <thead>
+          <tr>
+            <th scope="col">Subject</th>
+            <th scope="col">Bursary</th>
+            <th scope="col">Scholarship</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Biology</td>
+            <td>£20,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Chemistry</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Computing</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Design and technology</td>
+            <td>£20,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>English</td>
+            <td>£15,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Geography</td>
+            <td>£25,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Languages</strong><br> (French, German and Spanish only)</td>
+            <td>£25,000</td>
+            <td>£27,000</td>
+          </tr>
+          <tr>
+            <td>Languages</strong><br> (all other languages, including ancient languages)</td>
+            <td>£25,000</td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Maths</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+          <tr>
+            <td>Physics</td>
+            <td>£27,000</td>
+            <td>£29,000</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h3>Undergraduate bursaries</h3>
+
+      <p>You might be eligible for a bursary of £9,000 if you do:</p>
+
+      <ul>
+        <li>an undergraduate maths or physics course leading to qualified teacher status (QTS)</li>
+        <li>an opt-in undergraduate QTS course in computing, languages, maths or physics</li>
+      </ul>
+
+      <p><%= link_to("Find out more about how to get a bursary or scholarship", page_path("funding-and-support/scholarships-and-bursaries")) %>.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">More financial support</h2>
+
+    <div class="inset">
+      <p>
+        You can get a tuition fee and maintenance loan to fund your teacher training,
+        even if you've had student finance before.
+      </p>
+
+      <p>You could also be eligible for up to:</p>
```

Comment:
we don't specify these costs on the site @EllieNodder - is there a particularly strong reason to do so here?

Reply:
One of the comments left by the advertising team was if we could add a bit more info/the amount they receive as parent/carer ad copy works well for the over 30's, so it would be good to give them a short bit of info before they leave the page.

(I also do think we should look back over our parent/carer/disabled pages and potentially add more information in - I don't think the parent/carer copy is all that helpful at the minute, as it just has a list of links.)

But also happy to tweak back if you think necessary? @gemmadallmandfe @philspray

Reply:
one step ahead 😎 we've already got this one on the backlog https://trello.com/c/kF5WGOn3/1139-improve-if-youre-a-parent-or-carer-page